doc/TESTS.rst.txt

```diff
@@ -1,19 +1,18 @@
-.. -*- rest -*-
-
 NumPy/SciPy Testing Guidelines
 ==============================
 
 .. contents::
 
+
 Introduction
 ''''''''''''
 
-SciPy uses the `Nose testing system
-<http://nose.readthedocs.io>`__, with some
-minor convenience features added.  Nose is an extension of the unit
-testing framework offered by `unittest.py
-<http://docs.python.org/lib/module-unittest.html>`__. Our goal is that
-every module and package in SciPy should have a thorough set of unit
+Until the 1.15 release, NumPy based its test harnesses on the nose_
+framework. It now uses pytest_, but still
+supports pre-1.15 ``nose`` constructs for backwards compatibility.
```

Comment:
We support and test the nose constructs for downstream projects, but numpy tests should be strictly pytest.

Reply:
adopted alternative in the next comment